conanfile.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Conan recipe package for BehaviorTree.CPP
+"""
+from conans import ConanFile, CMake, tools
+
+
+class BehaviorTreeConan(ConanFile):
+    name = "BehaviorTree.CPP"
+    license = "MIT"
+    url = "https://github.com/BehaviorTree/BehaviorTree.CPP"
+    author = "Davide Faconti <davide.faconti@gmail.com>"
+    topics = ("conan", "behaviortree", "ai", "robotics", "games", "coordination")
+    description = "This C++ library provides a framework to create BehaviorTrees. It was designed to be flexible, easy to use and fast."
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "zmq": [True, False]}
+    default_options = {"shared": False, "zmq": True}
+    generators = "cmake"
+    exports = "LICENSE"
+    exports_sources = ("cmake/*", "include/*", "src/*", "3rdparty/*", "CMakeLists.txt")
```

Comment:
For the 3rdparty, it would be nice to be able to avoid it and use conan packages.
I think there are packages for flatbuffers and for tinyXML2.

This could be done later too, not critical, though it might save some building time too, and avoid conflicts for users using those libraries too.

Reply:
Indeed. If you notice: https://github.com/BehaviorTree/BehaviorTree.CPP/blob/master/CMakeLists.txt#L30

When ZMQ is detected the definition ZMQ_FOUND is added, otherwise, it isn't.

Reply:
both flatbuffers and tinyXML2 cpp files are compiled directly with BehaviorTree. We could change the cmake file to use both cpp files from Conan but could result in different ABI when building with/without Conan.